config.json

```diff
@@ -643,7 +643,13 @@
       "core": true,
       "difficulty": 6,
       "slug": "linked-list",
-      "topics": null,
+      "topics": [
+        "algorithms",
+        "logic",
+        "lists",
+        "integers",
+        "conditionals"
```

Comment:
These are good suggestions @ryan-neff :) I agree that `lists` and `algorithms` are good topics for this exercise. I would also add `generics` :)

I'm not sure about adding `logic`, `integers` and `conditionals` as topics. `logic` and `conditionals` are used in this exercise but they are used in most exercises and I wouldn't say that they're the main focus of this exercise. So it might be best to leave them out. `integers` are used but since using generics is partly the main focus of the exercise, you shouldn't need to think about integers at all when implementing it. Does that make sense? :)

Reply:
The config now has only algorithms, generics, and lists in its topics section f338ddd